client/src/components/ideas/IdeaDetails.vue

```diff
@@ -49,25 +49,8 @@
 
         <div class="view__form-element">
           <label class="view__label" for="create__type">Type</label>
-          <div class="view__radio-group">
-            <div class="view__radio view__option" v-bind:class="{ active: ideaTypeCode === PUBLIC_IDEA }" @mouseover="upHere = 0" @mouseleave="upHere = -1">
-              <input type="radio" name="ideatype" v-validate="'required'" value="0" v-model="ideaTypeCode" disable>
-              <div class="view__type-title tooltip">Public
-                <span class="view__type-desc tooltiptext" v-if="upHere == 0">Anyone can read and give feedback</span>
-              </div>
-            </div>
-            <div class="view__radio view__option" v-bind:class="{ active: ideaTypeCode === PRIVATE_IDEA }" @mouseover="upHere = 1" @mouseleave="upHere = -1">
-              <input type="radio" name="ideatype" value="1" v-model="ideaTypeCode" disable>
-              <div class="view__type-title tooltip">Private
-                <span class="view__type-desc tooltiptext" v-if="upHere == 1">Only visible to people who agree to the license</span>
-              </div>
-            </div>
-            <div class="view__radio view__option" v-bind:class="{ active: ideaTypeCode === COMMERCIAL_IDEA }" @mouseover="upHere = 2" @mouseleave="upHere = -1">
-              <input type="radio" name="ideatype" value="2" v-model="ideaTypeCode" disable>
-              <div class="view__type-title tooltip">Custom
-                <span class="view__type-desc tooltiptext" v-if="upHere == 2">Customise the license and choose who can see the idea</span>
-              </div>
-            </div>
+          <div class="view__input">
+            {{ ideaTypeCode === 0 ? 'Public' : ideaTypeCode === 1 ? 'Private' : 'Custom'}}
```

Comment:
Made these changes for resolution of merge conflict, Here I guess the numbers can be exchanged with the constant variables 👍